app-static/converter/revealmarkdownconverter.cpp

```diff
@@ -30,7 +30,8 @@ QString RevealMarkdownConverter::renderAsHtml(MarkdownDocument *document)
 
     if (document) {
         RevealMarkdownDocument *doc = dynamic_cast<RevealMarkdownDocument*>(document);
-        html = doc->markdownText;
+        if(doc)
```

Comment:
Thanks for the pull request. Just a minor style nitpick: 

Please add braces to the if-statement and a space between if and the bracket. Thank you!

BTW, could you also do the same change in `HoedownMarkdownConverter::renderAsHtml()`?